packages/ui-components/src/components/virtualized-list/virtualized-list.tsx

```diff
@@ -17,24 +17,18 @@ export class KvVirtualizedList implements IVirtualizedList {
 	/** @inheritdoc */
 	@Prop({ reflect: true }) getItemKey: (index: number) => string;
 	/** @inheritdoc */
-	@Prop({ reflect: true }) getItemHeight?: (index: number) => number;
-	/** @inheritdoc */
 	@Prop({ reflect: true }) renderItem: RenderItemFunc;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) overscanCount: number = 5;
```

Comment:
This value is a reference to the children inside of items?

Reply:
This is the number of items to render above/below the visible bounds of the list. This reduces the flickering during scrolling.